CorePayments/src/test/java/com/paypal/android/corepayments/api/AuthenticationSecureTokenServiceAPIUnitTest.kt

```diff
@@ -62,8 +63,8 @@ class AuthenticationSecureTokenServiceAPIUnitTest {
         assertEquals("application/x-www-form-urlencoded", headers["Content-Type"])
         assert(headers["Authorization"]!!.startsWith("Basic "))
```

Comment:
```suggestion
        assertTrue(headers["Authorization"]!!.startsWith("Basic "))
```
I noticed this is Java `assert` rather than a JUnit `assertTrue`.